ui/pages/institutional/custody/custody.js

```diff
@@ -597,8 +597,11 @@ const CustodyPage = () => {
                 history.push({
                   pathname: CUSTODY_ACCOUNT_DONE_ROUTE,
                   state: {
-                    imgSrc: selectedCustodian.iconUrl,
```

Comment:
@zone-live These changes were not in the PR that was cherry-picked into this branch. Are they supposed to be here? And if so is there any reason they are included here and not on develop?

Reply:
They need to be here as well, in develop they are [here](https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/institutional/custody/custody.js#L635) 👍🏼 
@danjm

Reply:
Instead of adding these changes on this branch, can we cherry-pick the commit that added them to develop?

Reply:
I thought about it, but it would introduce more changes that we don't want here for now doing that. It's just this bit that is needed.